src/components/DpAccordion/DpAccordion.stories.mdx

```diff
@@ -0,0 +1,28 @@
+import { Meta } from '@storybook/addon-docs'
+import DpAccordion from './DpAccordion'
+
+<Meta
+    title="Components/Accordion"
+    component={DpAccordion}
+/>
+
+# Accordion
+
```

Comment:
Maybe we can add a paragraph on when to use the accordion? Exhaustive [information on this](https://design-system.service.gov.uk/components/accordion/) can be found in the GOV.UK design system. In [the documentation on the `Details` component](https://github.com/demos-europe/demosplan-ui/blob/main/src/components/DpDetails/DpDetails.stories.mdx#details) i already mentioned the differentiation of accordion vs. details.

Reply:
At first I had the idea of adding basic documentation for every component that I move from the `core` to the `components` folder, but it slows down the process of cleaning up the components and we actually have a separate ticket for the documentation (https://yaits.demos-deutschland.de/T30976) which @muellerdemos wants to work on, so I decided to just move the files here.